src/nvim/api/deprecated.c

```diff
@@ -697,3 +698,18 @@ static void set_option_to(uint64_t channel_id, void *to, OptReqScope req_scope,
     set_option_value_for(name.data, opt_idx, optval, opt_flags, req_scope, to, err);
   });
 }
+
+/// @nodoc
+/// @deprecated
+void nvim__buf_redraw_range(Buffer buffer, Integer first, Integer last, Error *err)
```

Comment:
this is a private api, can we just remove it now?

Reply:
Fine by me but considering that it is used in the wild a single release cycle with `@deprecated` could be good?

Reply:
In general we should be pretty strict about not making efforts (deprecation cycles) for APIs that were marked as private. I see some important plugins in there, though I think it will be easy for them to migrate...